### PR TITLE
fix: user avatar new flow (AR-1275)

### PR DESCRIPTION
**Pull request by yamilmedina**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1275" title="AR-1275" target="_blank">StoryAR-1275</a>  Upload user profile image [new flow]
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [X] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [X] contains a reference JIRA issue number like `SQPIT-764`
  - [X] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [X] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

This fixes all the comments section pointed out on the ticket https://wearezeta.atlassian.net/browse/AR-1275 and ticket https://wearezeta.atlassian.net/browse/AR-1101

Basically visual adjustment and navigation behavior

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [X] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [X] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-467/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(2987954)(2987954)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(2987954)(2987954)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt [hunk 1]
```diff
@@ -109,7 +109,7 @@ enum class NavigationItem(
         primaryRoute = SELF_USER_PROFILE,
         canonicalRoute = "$SELF_USER_PROFILE/{$EXTRA_USER_ID}",
         content = { SelfUserProfileScreen() },
-        animationConfig = NavigationAnimationConfig.CustomAnimation(smoothSlideInFromRight(), smoothSlideOutFromLeft())
+        animationConfig = NavigationAnimationConfig.CustomAnimation(expandInToView(), shrinkOutFromView())
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt:112]: Somehow it feels like only the initial `expandIn` animation is happening and not the `shrinkOut`. Could you identify why?

https://user-images.githubusercontent.com/2468164/161495542-d9cbd08c-264c-4bce-a34f-3990150f2995.mov

**Review by gongracr — COMMENTED** [review]
Looking great. Just a small comment about improving the in/out animation style when opening/closing the SelfUserProfileScreen

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-467/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3009571)(3009571)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3009571)(3009571)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-467/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3009714)(3009714)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3009714)(3009714)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-467/4/) **failed**.

File: app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt (hunk 1 above)
Comment (yamilmedina) [reply app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt:112]: I'm was having a look during the morning, and it seems there is no easy solution. We have 2 `NavGraphs` that are conflicting for the animation, the `HomeNavGraph` (the one with conversations) is taking over the Items `NavGraph`.

So when we are exiting out from `SelfUserProfileScreen` we are just seeing the Home/Conversations animation, that currently doesn't have.

I think, I could try some orchestration with the enter animation of the Home/Conversations as a last resort

Comment (yamilmedina) [reply app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt:114]: @gongracr Fixed it, I was able to achieve it by setting the animations on HomeScreen to null instead of None.
Since None was overriding the animation spec of the parent also, as stated in the documentatio https://google.github.io/accompanist/navigation-animation/

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-467/5/) **failed**.

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-467/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3012696)(3012696)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3012696)(3012696)-dev-release.apk)

**Review by gongracr — APPROVED** [review]
Thanks for giving it an extra spin, now it looks much more Bacán, GJ! 🎯🎮🎊🪄
